boilerplate.js

```diff
@@ -116,9 +116,9 @@ async function install(context) {
     { template: 'index.js.ejs', target: 'index.js' },
     { template: 'README.md', target: 'README.md' },
     { template: '.gitignore.ejs', target: '.gitignore' },
+    { template: '.env.example', target: '.env' },
     { template: '.prettierignore', target: '.prettierignore' },
     { template: '.solidarity', target: '.solidarity' },
-    { template: '.babelrc', target: '.babelrc' },
```

Comment:
We added `.babelrc` back in because using package.json was causing this issue: https://github.com/infinitered/ignite-bowser/issues/145.